### ISSUE TITLE
Jenkins Plugin that allows us to upload Artifacts/Builds

### ISSUE THREAD

**Issue by aberhaneTopps** [microsoft/appcenter#234]
I would like to download a plugin in Jenkins that would allow me to continue my current Continuous Development Build Process. 

Currently a plugin exists for Hockeyapp and TestFairy that exceed our expectations and meet our goals.

**Comment by derpixeldan** [comment]
Hi, the HockeyApp plugin is a community contribution but there's a thread on Jenkins about supporting App Center: https://issues.jenkins-ci.org/browse/JENKINS-55013

**Comment by GinnWang** [comment]
Just to report an issue here that app Center plugin in Jenkins doesn't support relative path to app. We had configured a CI/CD pipeline in Jenkins to upload app files to Hockey app after a new build. When i try to replicate the same thing using app Center plugin, I found that this plugin doesn't support relative app path. It says 'io.jenkins.plugins.appcenter.AppCenterException: File not found' in the log.

**To Reproduce**
1. Configure a Jenkins job to build your app and then add a 'Post-build Actions' using App Center Plugins for Jenkins.
2. In 'Path to App' field, input a real path. E.g. ./artifacts/MyApp-1.0-Test.ipa
3. Trigger the build and the app is uploaded to App Center successfully.
4. Change the 'Path to App' to this : ./artifacts/*.ipa
5. Trigger this Jenkins Job. It failed because app cannot found.

**Expected behavior**
App Center plugin in Jenkins should support relative app path.

FYI I am using App Center plugin for Jenkins Version 0.2.0
